src/pages/journal/components/test-slot/test-slot.ts

```diff
@@ -54,11 +58,15 @@ export class TestSlotComponent implements SlotComponent, OnInit, OnDestroy {
         select(getTests),
         select(tests => getTestStatus(tests, slotId)),
       ),
+      testActivityCode$: this.store$.pipe(
+        select(getTests),
+        map(tests => getActivityCodeBySlotId(tests, this.slot.slotDetail.slotId)),
+      ),
     };
 
-    const { testStatus$ } = this.componentState;
+    const { testStatus$, testActivityCode$ } = this.componentState;
 
-    this.subscription = merge(testStatus$).subscribe();
+    this.subscription = merge(testStatus$, testActivityCode$).subscribe();
```

Comment:
Do we need this `.subscribe` if we are using the `| async` pipe in the template?

Reply:
Yep - hadn't realised they provide the same functionality so have removed